sale_order_line_description/README.rst

```diff
@@ -0,0 +1,100 @@
+===========================
+Sale order line description
+===========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/11.0/sale_order_line_description
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-11-0/sale-workflow-11-0-sale_order_line_description
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module sets to use only product's sale description field on the sale order
+lines, instead of "Ref+Name+Sale Description".
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+The user has to belong to group_use_product_description_per_so_line.
+This is possible by selecting the related option in the following menu:
+
+* *Sales > Settings > Quotations & Orders > Product Sale Description*
+
+Usage
+=====
+
+#. Go to **Sales > Settings > Quotations & Orders** and check "Product Sale
+   description* to use only product sale description on the sales order lines.
+#. Go to **Setting > Users & Companies > Users** and check you have checked
+   the "Use only product purchase description on order lines" access right for
+   your user.
+#. Add or modify a sale description to any of your products.
+#. Create a sale order and add a sale order line with this product and check
+   out that now sale order line description only contains the description you
+   are created.
```

Comment:
`you are created`: could you please rephrase it?

Reply:
@tafaRU can you recheck now?

Reply:
Ok, now the `USAGE` section LGTM. Thanks!